workers/unity/Packages/io.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -245,6 +230,33 @@ private static void Generate()
             }
         }
 
+        private static string GetSchemaCompilerPath()
+        {
+            var schemaCompilerPath = SchemaCompilerPath;
+
+            switch (Application.platform)
+            {
+                case RuntimePlatform.WindowsEditor:
+                    schemaCompilerPath = Path.ChangeExtension(schemaCompilerPath, ".exe");
+                    break;
+                case RuntimePlatform.OSXEditor:
+                    var result = RedirectedProcess.Command("chmod")
+                        .WithArgs("+x", $"\"{schemaCompilerPath}\"")
+                        .InDirectory(Path.GetFullPath(Path.Combine(Application.dataPath, "..")))
+                        .Run();
+                    if (result.ExitCode != 0)
+                    {
+                        throw new IOException($"Failed to chmod schema compiler:\n{string.Join("\n", result.Stderr)}");
```

Comment:
`\n` -> `Environment.Newline` ?

Reply:
we use `\n` everywhere else atm so... nah